cmd/gaia/testnets/README.md

```diff
@@ -159,70 +98,70 @@ Start the full node:
 gaiad start
 ```
 
-Check the everything is running smoothly:
+Check that everything is running smoothly:
 
 ```
 gaiacli status
 ```
 
-## Generate keys
+View the status of the network with the [Cosmos Explorer](https://explorecosmos.network).Once your full node syncs up to the current block height, you should see it appear on the [list of full nodes](https://explorecosmos.network/validators).
+
+## Generate Keys
 
 You'll need a private and public key pair \(a.k.a. `sk, pk` respectively\) to be able to receive funds, send txs, bond tx, etc.
 
-To generate your a new key \(default _ed25519_ elliptic curve\):
+To generate a new key \(default _ed25519_ elliptic curve\):
 
 ```
 gaiacli keys add <your_key_name>
 ```
 
-Next, you will have to enter a passphrase for your key twice. Save the _seed_ _phrase_ in a safe place in case you forget the password.
+Next, you will have to create a passphrase. Save the _seed_ _phrase_ in a safe place in case you forget the password.
 
-Now if you check your private keys you will see the `<your_key_name>` key among them:
+If you check your private keys, you'll now see `<your_key_name>`:
 
 ```
 gaiacli keys show <your_key_name>
 ```
 
-You can see all your other available keys by typing:
+You can see all your available keys by typing:
 
 ```
 gaiacli keys list
 ```
 
```

Comment:
Usually, newly joining validators would query their accounts right after they create the key and they all ask about the `panic` error message. Better add this and let them have expectation on seeing this rather than keep asking every time.

Reply:
The panic is fixed in the latest version of gaiad!

Reply:
Oh sorry. Didn't notice this.

Reply:
Thanks, I just added a note about querying an empty account.

Reply:
Just tried and it becomes an error message. 

Would it be still good to put the error message in? I think most users don't know what's happening with this.

`ERROR: Error{1:9,--= Error =--`
`Message: No account with address cosmosaccaddr14c830n7ew3awdk4n770j9ckw3x4dlj6nwfd0ne was found in the state.`
`Are you sure there has been a transaction involving it?`
`Cause: <nil>`
`T: 0x9`
`Msg Traces:`
`--= /Error =--`
`}`

Reply:
Yeah, I'll put a section of that error message in the readme